src/Microsoft.Data.Analysis/Strings.Designer.cs

```diff
@@ -420,6 +411,15 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Index cannot be greater than the Column&apos;s Length.
```

Comment:
Column? "///   Looks up a localized string similar to Index cannot be greater than the Column&apos;s Length."

Reply:
Right, in this case Column consists of rows. Length of each column in the dataframe should be equal to RowCount. When we indexing for each row, we shouldn't exceed length of the column (or DataFrame RowCount if column is attached to a dataframe). Maybe for resource it's not the best name, I may discard the chane. However everywhere in our DataFrameColumn API we use `rowIndex` as a parameter name (not just index).

```
/// <summary>
/// Indexer to get/set values at <paramref name="rowIndex"/>
/// </summary>
/// <param name="rowIndex">The index to look up</param>
/// <returns>The value at <paramref name="rowIndex"/></returns>
public object this[long rowIndex]
{
    get => GetValue(rowIndex);
    set => SetValue(rowIndex, value);
}
```
and code like this looks suspiciously in case of using name ColumnIndexOutOfRange for the resource. 

```
private int GetBufferIndexContainingRowIndex(long rowIndex)
{
    if (rowIndex >= Length)
    {
        throw new ArgumentOutOfRangeException(Strings.ColumnIndexOutOfRange, nameof(rowIndex));
    }

    return (int)(rowIndex / MaxCapacity);
}
```

We can rename rowIndex to just index everywhere in out API, so ColumnIndexOutOfRange will not look ambiguous and will mean index that is used for indexing column. 

What is your opinion on this?

Reply:
Renamed to _IndexIsGreaterThanColumnLength_ as it closer to the resource content